src/test/cypress/integration/exam/ExamAssessment.spec.ts

```diff
@@ -124,11 +130,64 @@ describe('Exam assessment', () => {
         });
     });
 
+    describe('Assess a quiz exercise submission', () => {
+        let resultDate: Dayjs;
+
+        beforeEach('Generate new exam name', () => {
+            examEnd = dayjs().add(15, 'seconds');
+            resultDate = examEnd.add(10, 'seconds');
+            prepareExam(examEnd, resultDate);
+        });
+
+        beforeEach('Create exercise and submission', () => {
+            courseManagementRequests.createQuizExercise({ exerciseGroup }, [multipleChoiceQuizTemplate], 'Cypress Quiz').then(() => {
+                courseManagementRequests.generateMissingIndividualExams(exam);
+                courseManagementRequests.prepareExerciseStartForExam(exam);
+                cy.login(student, '/courses/' + course.id + '/exams/' + exam.id);
+                examStartEnd.startExam();
+                cy.contains('Cypress Quiz').click();
+                cy.get('#answer-option-0 > .selection > .ng-fa-icon > .svg-inline--fa').click();
+                cy.get('#answer-option-2 > .selection > .ng-fa-icon > .svg-inline--fa').click();
+                cy.get('#exam-navigation-bar').find('.btn-danger').click();
```

Comment:
There is a `ExamNavigationBar` page object, which exposes the '#exam-navigation-bar' selector. I assume you want to hand in early here. The `ExamNavigationBar` already has a function for that